### ISSUE TITLE
Could async_resume_with be added?

### ISSUE THREAD

**Issue by dignifiedquire** [whatisaphone/genawaiter#21]
I was trying to call `async_resume_with` but couldn't, and looked at the docs and it does not exist. Any specific reason why?

**Comment by whatisaphone** [comment]
Good catch, this is just an oversight, there's no reason it couldn't exist.

I'd accept a PR for this, otherwise I'll try to get to it myself at some point.
